src/sentry/backup/comparators.py

```diff
@@ -344,7 +367,10 @@ def build_default_comparators():
             "sentry.sentryapp": [EmailObfuscatingComparator("creator_label")],
             "sentry.servicehook": [HashObfuscatingComparator("secret")],
             "sentry.user": [HashObfuscatingComparator("password")],
-            "sentry.useremail": [HashObfuscatingComparator("validation_hash")],
+            "sentry.useremail": [
+                DateUpdatedComparator("date_hash_added"),
+                IgnoredComparator("validation_hash", "is_verified"),
```

Comment:
Just to confirm, should `is_verified` also be an `IgnoredComparator`?

Reply:
Yeah. The idea here is that many users can share a single useremail (apparently Sentry allows multiple users per email account). So when importing, we will have special handling for `UserEmail`, using `.get_or_create()` instead of the usual `.save(force_insert=True)`. The upshot is that if a `useremail` matching one from your import already exists, you will just defer to its contents.

For your example specifically: if a useremail already exists with `is_verified = True`, we want to make sure we don't overwrite that with our import. As the comment on `IgnoredComparator` says, anything you use it on should be verified by other means, so we'll probably need to have logic that ex ensures `is_verified` is always set to `False` on imported useremails.